src/engine/Engine.ts

```diff
@@ -81,32 +58,25 @@ export enum ScrollPreventionMode {
   All
 }
 
-/**
- * Interface describing the absolute CSS position of the game window. For use when [[DisplayMode.Position]]
- * is specified and when the user wants to define exact pixel spacing of the window.
- * When a number is given, the value is interpreted as pixels
- */
-export interface AbsolutePosition {
-  top?: number | string;
-  left?: number | string;
-  right?: number | string;
-  bottom?: number | string;
-}
-
 /**
  * Defines the available options to configure the Excalibur engine at constructor time.
  */
 export interface EngineOptions {
   /**
-   * Optionally configure the native canvas width of the game
+   * Optionally configure the native logical pixel width of the game (resolution)
```

Comment:
Might be nice to include symbol link to `Resolution`

```suggestion
   * Optionally configure the native logical pixel width of the game. See [[Resolution]] for common presets.
```